src/main/java/com/google/cloud/tools/gradle/appengine/appyaml/AppEngineAppYamlPlugin.java

```diff
@@ -34,36 +34,36 @@
 import org.gradle.api.tasks.bundling.Jar;
 import org.gradle.api.tasks.bundling.War;
 
-/** Plugin definition for App Engine flexible environments. */
-public class AppEngineFlexiblePlugin implements Plugin<Project> {
+/** Plugin definition for App Engine app.yaml based projects. */
+public class AppEngineAppYamlPlugin implements Plugin<Project> {
 
-  public static final String APP_ENGINE_FLEXIBLE_TASK_GROUP = "App Engine Flexible environment";
+  public static final String APP_ENGINE_APPYAML_TASK_GROUP = "App Engine app.yaml based projects";
```

Comment:
nit: do you think `APPYAML` is better than `APP_YAML`?

Reply:
APP_YAML is probably right choice